src/main/java/hw7/entities/DataEntry.java

```diff
@@ -0,0 +1,65 @@
+package hw7.entities;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import lombok.Getter;
+import lombok.ToString;
+import org.json.JSONArray;
+
+@Getter
+@ToString
+public class DataEntry {
```

Comment:
What kind of data is this ? Class name should be written with reference to it business purpose